packages/create/src/recipes/sveltekit/template/ts/src/hooks.server.__auth.ts

```diff
@@ -0,0 +1,45 @@
+import { redirect, type Handle } from "@sveltejs/kit";
+import { sequence } from "@sveltejs/kit/hooks";
+import serverAuth, {
+  type AuthRouteHandlers,
+} from "@edgedb/auth-sveltekit/server";
+import { client } from "$lib/server/auth";
+import { options } from "$lib/auth";
+
+const authRouteHandlers: Partial<AuthRouteHandlers> = {
+  async onBuiltinUICallback({ error, tokenData, isSignUp }) {
+    if (error) {
+      //
+    }
+
+    if (!tokenData) {
+      //
+    }
+
+    if (isSignUp) {
+      //
+    }
+
+    redirect(303, "/");
+  },
+
+  async onSignout() {
+    redirect(303, "/");
+  },
+};
+
+const { createServerRequestAuth, createAuthRouteHook } = serverAuth(
+  client,
+  options
+);
+
+const createServerAuthClient: Handle = ({ event, resolve }) => {
```

Comment:
Can we inline this function in the `sequence` call? It avoids needing to come up with a name for this function. 😄

Reply:
I think it can be cleaner if function is separate but if u think name is not the best I can inline it.